index.js

```diff
@@ -33,9 +36,24 @@ module.exports = async function (
       continue;
     }
 
+    if (onlyOwner && actions[i].owner !== onlyOwner ){
```

Comment:
Please extract this in to another file (e.g. `checkSelectOnly.js`) and write unit tests.

Example module: https://github.com/mheap/pin-github-action/blob/main/checkAllowedRepos.js
Example tests: https://github.com/mheap/pin-github-action/blob/main/checkIgnoredRepos.test.js